docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+# Enabling the OpenStack CLI tool
+
+The OpenStack CLI tool, also known as OpenStack Command Line Client, 
+conveniently provides access to various OpenStack APIs. Using the 
+OpenStack CLI tool, you can remotely create and manage the lifecycle of 
+objects relating, for example, to Compute, Networking, and Storage.
+
+Before installing `openstack` to your local laptop or workstation, you 
+first need to have an OpenStack user in your {{extra.brand}} account. 
+Next, you create and download a special RC file onto your computer, 
+modify it to reflect your OpenStack user's credentials, and source it. 
+Only then will you be able to use `openstack` --- provided it is 
+installed, of course.
+
+## Creating an OpenStack user
+
+From your favorite web browser, navigate to the [Cleura 
+Cloud](https://{{extra.gui_domain}}) page, and login into your 
+{{extra.brand}} account.
+
+Please make sure the left-hand side pane on the {{extra.gui}} is fully 
+visible, click the _Users_ category to expand it, and click on 
+_Openstack Users_.
```

Comment:
It really bugs me that the Control Panel botches the capitalization here and says "Openstack", rather than "OpenStack". But I agree that when we describe GUI elements, we should retain their spelling, so let's leave this as-is.

Reply:
That inconsistency has been haunting me since yesterday. I am not joking.